packages/components/nodes/chatmodels/ChatOpenAI/ChatOpenAI.ts

```diff
@@ -152,6 +152,73 @@ class ChatOpenAI_ChatModels implements INode {
                 type: 'json',
                 optional: true,
                 additionalParams: true
+            },
+            {
+                label: 'Allow Image Uploads',
```

Comment:
Maybe have a description saying it will automatically use gpt-vision when image is uploaded in the chat